docs/modules/ROOT/pages/tutorials/getting-started/java-terminal.adoc

```diff
@@ -0,0 +1,310 @@
+
+. Login to the {product} console with your web browser.
+. Click on your user name on the top right and select "Copy login command"
+. Click "Display token" and copy the login command shown in "Log in with this token"
+. Paste the `oc login` command on the terminal:
++
+[source,shell]
+--
+$ oc login --token=sha256~_xxxxxx_xxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxx-X --server=https://api.[YOUR_PREFERRED_ZONE].appuio.cloud:6443
+$ oc projects
+You aren't a member of any projects. You can request a project to be created with the 'new-project' command.
+--
+
+. Create a new project called "fortune-java"
++
+[source,shell]
+--
+$ oc new-project fortune-java
+Now using project "fortune-java" on server "https://api.[YOUR_PREFERRED_ZONE].appuio.cloud:6443".
+
+You can add applications to this project with the 'new-app' command. For example, try:
+
+    oc new-app rails-postgresql-example
+
+to build a new example application in Ruby. Or use kubectl to deploy a simple Kubernetes application:
+
+    kubectl create deployment hello-node --image=k8s.gcr.io/serve_hostname
+--
+
+. To deploy the application we will use a standard Kubernetes `Deployment` object. Save the following YAML in a file called `deployment.yaml`:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fortune-java
+  namespace: fortune-java # <1>
+  labels:
+    app: fortune-java
+spec:
+  template:
+    spec:
+      containers:
+      - image: registry.gitlab.com/vshn/applications/fortune-java:latest
+        imagePullPolicy: Always
+        name: fortune-container
+        ports:
+        - containerPort: 8080
+    metadata:
+      labels:
+        app: fortune-java
+  selector:
+    matchLabels:
+      app: fortune-java
+  strategy:
+    type: Recreate
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fortune-java
+  namespace: fortune-java # <1>
+  labels:
+    app: fortune-java
+spec:
+  ports:
+    - port: 8080
+      targetPort: 8080
+  selector:
+    app: fortune-java
+  type: ClusterIP
+----
+<1> Make sure this annotation matches exactly the name of your project: `fortune-java`
+
+. Then apply the deployment to your {product} project and wait until your pod appears with the status "Running":
++
+[source,shell]
+--
+$ oc -n fortune-java apply -f deployment.yaml
+deployment.apps/fortune-java created
+service/fortune-java created
+$ oc -n fortune-java get pods --watch
+NAME                         READY   STATUS    RESTARTS   AGE
+fortune-java-6fbd5484cf-k47gt   1/1     Running   0          11s
+--
+
+== Step 2: Publish your Application
+
+At the moment your container is running but it's not available from the Internet. To be able to access our application, we must create an `Ingress` object.
+
+. Create another file called `ingress.yaml` with the following contents, customizing the parts marked as `[YOUR_APP_NAME]` and `[YOUR_PREFERRED_ZONE]` to your liking:
```

Comment:
Maybe point to the zone documentation to find the exact information how to form the `host` name? I wouldn't be able to understand how to construct it withouth knowing how the zone is named...

Reply:
Added reference to docs